src/network/networktask.cpp

```diff
@@ -66,6 +66,25 @@ void NetworkTask::emitAborted(
     emit aborted(requestUrl);
 }
 
+void NetworkTask::emitNotFound(QNetworkReply::NetworkError errorCode,
```

Comment:
Can we avoid this extra signal? Is Webtask::onNetworkError() sufficient?

Reply:
As far as I remember, onNetworkError was related with the TagFetcher, so when this signal is emitted, Tag Fetcher Dialog refreshes itself, so the results from MusicBrainz were lost. I am not sure if this can be handled from slot of onNetworkError.